msteams-platform/mstdd-landing.yml

```diff
@@ -0,0 +1,94 @@
+metadata:
+  title: Microsoft Teams Developer Documentation
+  description: Introducing building apps for Teams
+  ms.date: 11/10/2021
+  ms.topic: landing-page
+landingContent:
+- title: Explore new features
+  linkLists:
+    - linkListType: concept
+      links:
+      - text: Monetize your Teams app with a transactable SaaS offer
+        url: ~/concepts/deploy-and-publish/appsource/prepare/include-saas-offer.md
+      - text: Create low-code custom apps
+        url: ~/samples/teams-low-code-solutions.md
+- title: Tutorials
+  linkLists:
+    - linkListType: concept
+      links:
+        - text: Proactive messages
+          url: ~/bots/how-to/conversations/send-proactive-messages.md
+        - text: Add authentication to bots
+          url: ~/bots/how-to/authentication/auth-aad-sso-bots.md
+    - linkListType: tutorial
+      links:
+        - text: Step-by-step guide for Proactive messages
+          url: ~/sbs-send-proactive.yml
+        - text: Step-by-step guide for adding authentication to bots
+          url: ~/sbs-bots-with-sso.yml
+- title: Get started
+  linkLists:
+    - linkListType: get-started
+      links:
+      - text: Get started with building apps
+        url: ../msteams-platform/get-started/get-started-overview.md
+    - linkListType: tutorial
+      links:
+      - text: Build app with JavaScript
+        url: sbs-gs-javascript.yml
+      - text: Build app with Blazor
+        url: sbs-gs-blazor.yml
+      - text: Build app with SPFx
+        url: sbs-gs-spfx.yml
+      - text: Build app with C# or .NET
+        url: sbs-gs-csharp.yml
+      - text: Build app with Node.js
+        url: sbs-gs-nodejs.yml
+- title: About building apps for Teams
+  linkLists:
+    - linkListType: overview
+      links:
+      - text: Build apps for Microsoft Teams
+        url: ../msteams-platform/overview.md
+      - text: App fundamentals
+        url: ~/concepts/app-fundamentals-overview.md
+      - text: Design your app
+        url: ~/concepts/design/design-teams-app-overview.md
+    - linkListType: whats-new
+      links:
+      - text: What's new
+        url: ../msteams-platform/whats-new.md
+- title: Build app capabilities
+  linkLists:
+  - linkListType: concept
+    links:
+    - text: Build tabs
+      url: ~/tabs/what-are-tabs.md
+    - text: Build bots
+      url: ~/bots/what-are-bots.md
+    - text: Build message extensions
+      url: ~/messaging-extensions/what-are-messaging-extensions.md
+    - text: Build apps for Teams meetings
+      url: ~/apps-in-teams-meetings/teams-apps-in-meetings.md
+    - text: Build webhooks and connectors
+      url: ~/webhooks-and-connectors/what-are-webhooks-and-connectors.md
+    - text: Build cards and task modules
+      url: ~/task-modules-and-cards/cards-and-task-modules.md
+    - text: App manifest
+      url: ../msteams-platform/resources/schema/manifest-schema.md
+- title: Publish and Distribute
```

Comment:
Please change to:

Publish and distribute

Reply:
fixed